File: Arrays/ValidAnagram/validAnagram.go
func isAnagram(s string, t string) bool {
	sCounts := charCounts(s)
	tCounts := charCounts(t)
	for i, _ := range sCounts {
		if sCounts[i] != tCounts[i] {
			return false
		}
	}
	return true
}

func charCounts(word string) [26]int {
	result := [26]int{}
	for _, c := range word {
		index := int(rune(c) - rune('a'))
		result[index] = result[index] + 1
	}
	return result
}

File: Arrays/ValidAnagram/validAnagram.py
class Solution(object):
    def isAnagram(self, s, t):
        sCounts = self.charCount(s)
        tCounts = self.charCount(t)
        for i in range(len(sCounts)):
            if sCounts[i] != tCounts[i]:
                return False
        return True
    
    def charCount(self, word):
        result = [0] * 26
        for c in word:
            result[ord('a') - ord(c)] += 1
        return result
        